sirepo/pkcli/test_http.py

```diff
@@ -152,6 +152,8 @@ async def _sims_tasks():
                 s.append(
                     _Sim(a, e.name, r.report, r.binary_data_file).create_task(),
                 )
+                # Sleep a bit to not completely flood server
+                await tornado.gen.sleep(1)
```

Comment:
shouldn't the server just queue?

Reply:
It does but nginx in dev was just closing the connection when too many requests came in too quickly

https://github.com/radiasoft/sirepo/pull/3560/commits/f1f9b1aa4be4c77396e3467d5038c7b700e3997a